tests_end_to_end/tests/Datasets/test_dataset_items_crud_operations.py

```diff
@@ -12,7 +12,7 @@
 
 class TestDatasetItemsCrud:
     
-    @pytest.mark.parametrize('dataset_insert', ['insert_via_ui', 'insert_via_sdk'])
+    @pytest.mark.parametrize('dataset_insert', ['insert_via_ui'])
```

Comment:
Why is the parameter `insert_via_sdk` removed?

Reply:
i'm not actually sure, i thing i was testing something during development and forgot about it, added it back for proper coverage